### DISCUSSION TITLE
Legacy Console Edition Crafting

### DISCUSSION THREAD

**Discussion by waffleeiron** [BedrockTweaks/Files#386] [Ideas]
Some aspects of it might not be possible because of controller inputs but at least the gui of the legacy edition crafting

**Comment by UnderboladoTB** [comment]
Oh my peak Legacy Edition

**Comment by SquatchHunter** [comment]
Community packs will be considered after addons are released
and also I can already tell this is a pack no one will want to tackle _quickly_ as it will require a lot of JSON UI and we all hate it.
